services-directions-models/src/test/resources/route_options_v5.json

```diff
@@ -32,6 +32,7 @@
   "depart_at": "2021-02-02'T'02:02",
   "max_height": 1.5,
   "max_width": 1.4,
+  "max_weight": 2.9,
```

Comment:
Could you make this same change in https://github.com/mapbox/mapbox-java/blob/main/services-directions/src/test/resources/route_options_v5.json?

Reply:
nice catch. Added